resources/views/pages/trails/legacy.blade.php

```diff
@@ -0,0 +1,136 @@
+@extends('layouts.app')
+
+@section('title', 'About eBikes Florida')
+
+@section('description',
+    'Explore the benefits of electric bikes in Florida. Our site provides information and
+    resources.')
+
+@section('content')
+    {{-- @todo make this work in dark mode --}}
```

Comment:
This page looks like it works in dark mode. If so can you remove this comment?

Reply:
Updated